docs/guide/queries.md

```diff
@@ -1 +1,71 @@
 # Queries
+
+## Definition:
+
+Queries are dependencies to an async source of data. They allow us to declaratively fetch, cache and refresh the data.
+
+::: info
+Queries are meant to **read** data. In terms of REST, for example, queries would handle `GET` requests. If you need to **write** (or mutate) data, you can use [mutations](./mutations.md).
+:::
+
+## Defining a query:
+
+The API to define a query is the `useQuery` composable:
+
+```vue twoslash
+<script setup lang="ts">
+import { useQuery } from '@pinia/colada'
+
+const { data, error, isFetching } = useQuery({
+  key: ['todos'],
+  query: () => fetch('/api/todos').then((res) => res.json())
+})
+</script>
+
+<template>
+  <main>
+    <div v-if="isFetching">
```

Comment:
let's use the `status` instead, it should be the recommended approach as it covers more cases while `isFetching` is just a convenient wrapper